src/targets/gpu/fuse_mlir.cpp

```diff
@@ -373,6 +381,15 @@ bool is_standalone_convs_enabled(context* ctx)
     }
     return is_requested("convolution");
 }
+
+bool is_standalone_dots_enabled()
+{
+    if(is_self_decide())
+    {
+        return false;
+    }
+    return is_requested("dot");
+}
```

Comment:
Just make a `is_standalone_op_enabled` that takes the op name as a parameter.

Reply:
@pfultz2. Maybe, instead of `is_standalone_op_enabled`, we can generalize it for all ops and call it, for example, `is_op_enabled`?

What do you think about the following code?

```cpp
bool is_op_enabled(std::string_view op_name, context* ctx)
{
    if(is_self_decide())
    {
        if(op_name == "fused")
        {
            return true;
        }
        else if(op_name == "convolution")
        {
            if(ctx == nullptr)
            {
                return false;
            }
            else
            {
                const auto& device = ctx->get_current_device();
                const std::string navi_family{"gfx110"};
                return starts_with(device.get_gfx_name(), navi_family);
            }
        }
        else
        {
            return false;
        }
    }
    return is_requested(op_name);
}

} // namespace

#endif // MIGRAPHX_MLIR

void fuse_mlir::apply(module_pass_manager& mpm) const
{
#ifdef MIGRAPHX_MLIR
    if(is_op_enabled("fused", this->ctx))
    {
        match::find_matches(mpm, find_mlir_fused_ops{});
    }

    if(is_op_enabled("convolution", this->ctx))
    {
        match::find_matches(mpm, find_mlir_standalone_convolution_op{});
    }

    if(is_op_enabled("dot", this->ctx))
    {
        match::find_matches(mpm, find_mlir_standalone_dot_op{});
    }
#else
    (void)mpm;
#endif
}
```

Reply:
Yea thats a good idea.

Reply:
Thanks! Done.